### PR TITLE
:memo: Rename components to fit common naming pattern

### PR DESCRIPTION
**Pull request by stevenbal**
requires https://github.com/VNG-Realisatie/vng-api-common/pull/161 to be merged and version to be bumped first

### CODE REVIEW

File: src/zrc/api/schema.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 from .kanalen import KANAAL_ZAKEN
 
-description = f"""Een API om een zaakregistratiecomponent (ZRC) te benaderen.
+description = f"""Een API om een Zaken API (ZRC) te benaderen.
```
Comment (stevenbal) [inline src/zrc/api/schema.py:8]: @joeribekker is this replacement correct?

Comment (joeribekker) [reply src/zrc/api/schema.py:8]: nee deze niet, goed opgemerkt :)

Comment (stevenbal) [reply src/zrc/api/schema.py]: heb hem teruggedraaid

**Comment by stevenbal** [comment]
@joeribekker I double checked, and I used `vng-api-common==1.0.54` to generate the schema

**Review by joeribekker — APPROVED** [review]

**Pull request merged**